UPGRADE.md

```diff
@@ -10,6 +10,8 @@
 
 ### Core / CoreBundle
 
+* Renamed ``getLastNewPayment()`` on ``OrderInterface`` to ``getLastPayment($state)``, where ``$state`` is target last payment state.
```

Comment:
You can also add how exactly can I replace `getLastNewPayment()` in my userland code